scanpipe/pipes/scancode.py

```diff
@@ -231,12 +231,13 @@ def scan_for_package_data(location, with_threading=True, **kwargs):
     return _scan_resource(location, scanners, with_threading=with_threading)
 
 
-def save_scan_file_results(codebase_resource, scan_results, scan_errors):
+def save_scan_file_results(codebase_resource, scan_results, scan_errors, status=None):
     """
     Save the resource scan file results in the database.
     Create project errors if any occurred during the scan.
     """
-    status = flag.SCANNED
+    if not status:
+        status = flag.SCANNED
```

Comment:
I'm not sure about adding complexity and various status here. @pombredanne ?

Reply:
fixed